### DISCUSSION TITLE
Repository?

### DISCUSSION THREAD

**Discussion by JyanJohn** [Yisus7u7/termux-desktop-xfce#64] [General]
Hello, how did you get your own package repo? Ive been trying to figure it out for some time now

**Comment by JesusChapman** [comment]
> Hello, how did you get your own package repo? Ive been trying to figure it out for some time now

using termux-apt-repo, termux-create-package, and Github pages.

**Reply by JyanJohn** [reply]
can you teach me how to create my own repo for Termux?
